File: DSA/Data_Structure/Matrix/Easy/Richest_Customer_Wealth/Solutions/solve.java
package DSA.DSA.Data_Structure.Matrix.Easy.Richest_Customer_Wealth.Solutions;

class Solution {
    public static int maximumWealth(int[][] accounts) {
        int res = 0;
        for(int i = 0; i < accounts.length; i++){
            int total = 0;
            for(int j = 0; j < accounts[i].length; j++){
                total += accounts[i][j];
            }
            res = Math.max(res, total);
        }
        return res;
    }

    public static void main(String... args) {
        int[][] arr = {{1,5},{7,3},{3,5}};
        int op = maximumWealth(arr);
        System.out.println(op);
    }
}

File: DSA/Data_Structure/Matrix/Easy/Richest_Customer_Wealth/Solutions/solve.go
package main

import "fmt"

func maximumWealth(accounts [][]int) int {
    var res int = 0
    for i:= 0; i < len(accounts); i++ {
        var total int = 0
        for j := 0; j < len(accounts[i]); j++ {
            total += accounts[i][j]
        }
        if res < total {
            res = total
        }
    }
    return res
}

func main() {
    arr := [][]int{{1,2,3},{3,2,4}}
    fmt.Print(maximumWealth(arr))
}
